staker/staker.go

```diff
@@ -358,7 +358,7 @@ func (s *Staker) Act(ctx context.Context) (*types.Transaction, error) {
 	effectiveStrategy := s.strategy
 	nodesLinear, err := s.validatorUtils.AreUnresolvedNodesLinear(callOpts, s.rollupAddress)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error checking for rollup assertion fork: %w", err)
```

Comment:
add s.rollupAddress

Reply:
We can figure that out based on which chain the validator is running on